Alignment/OfflineValidation/test/PVValidation_T_cfg.py

```diff
@@ -0,0 +1,311 @@
+                                           VertexCollectionTag = cms.InputTag("VERTEXTYPETEMPLATE"), 
+                                           Debug = cms.bool(False),
+                                           isLightNtuple = cms.bool(True),
+                                           storeNtuple = cms.bool(False),
+                                           useTracksFromRecoVtx = cms.bool(False),
+                                           askFirstLayerHit = cms.bool(False),
+                                           probePt = cms.untracked.double(PTCUTTEMPLATE),
+                                           runControl = cms.untracked.bool(RUNCONTROLTEMPLATE),
+                                           runControlNumber = cms.untracked.uint32(int(runboundary)),
+                                           
+                                           TkFilterParameters = cms.PSet(algorithm=cms.string('filter'),                             
+                                                                         maxNormalizedChi2 = cms.double(5.0),                        # chi2ndof < 20                  
+                                                                         minPixelLayersWithHits=cms.int32(2),                        # PX hits > 2                   
+                                                                         minSiliconLayersWithHits = cms.int32(5),                    # TK hits > 5                   
+                                                                         maxD0Significance = cms.double(5.0),                        # fake cut (requiring 1 PXB hit)
+                                                                         minPt = cms.double(0.0),                                    # better for softish events     
+                                                                         trackQuality = cms.string("any")
+                                                                         ),
+                                        
+                                           TkClusParameters = cms.PSet(algorithm   = cms.string('gap'),
+                                                                       TkGapClusParameters = cms.PSet(zSeparation = cms.double(0.2)  # 0.2 cm max separation betw. clusters
+                                                                                                      ) 
+                                                                       )
+                                           )
+
+####################################################################
+# Path
+####################################################################
+process.p = cms.Path(process.goodvertexSkim*
+                     process.offlineBeamSpot*
+                     process.seqTrackselRefit*
+                     # in case the navigation shool is removed
+                     #process.MeasurementTrackerEvent*
```

Comment:
@mmusich I see now, that you want to keep the option to quickly enable the NavigationSchool again, i.e. you might ignore my comments above about the commented code